.github/actions/test_ya/action.yml

```diff
@@ -258,8 +259,37 @@ runs:
     
         .github/scripts/tests/split-junit.py -o "$JUNIT_REPORT_PARTS" "$JUNIT_REPORT_XML"
     
+    - name: Test history upload results to YDB
+      if: always()
+      continue-on-error: true
+      shell: bash
+      env:
+        PR_NUMBER: ${{ github.event.number }}
+        JOB_NAME: ${{ github.workflow }}
+        JOB_ID: ${{github.run_id }}
+      run: |
+        set -x
+        python3 -m pip install ydb ydb[yc]
+
+        # Debug variables
+        echo "JUNIT_REPORT_XML=${JUNIT_REPORT_XML}"
+        echo "BUILD_PRESET=${BUILD_PRESET}"
+        echo "PR_NUMBER=${PR_NUMBER}"
+        echo "JOB_NAME=${JOB_NAME}"
+        echo "JOB_ID=${JOB_ID}"
+        echo "GITHUB_REF_NAME=${GITHUB_REF_NAME}"
+        
+        script=".github/scripts/upload_tests_results.py --result-file ${JUNIT_REPORT_XML} --run-time $(date +%s) --commit $(git rev-parse HEAD) --build-type ${BUILD_PRESET} --pull ${PR_NUMBER} --job-name ${JOB_NAME} --job-id ${JOB_ID} --branch ${GITHUB_REF_NAME}"
+
+        printf "$script"
+        
+        # Attempt to run the script
+        result=$(python3 $script)
+        printf "$result"
```

Comment:
In other python3 calls in the current file an interpreter is not specified

Lets invoke python3 without specifying an interpreter